src/components/Genome3D/List/index.js

```diff
@@ -84,6 +86,15 @@ export const List = (
   customLocation: {search?: Object}} */,
 ) => {
   if (data.loading) return <Loading />;
+  if (!data.payload)
```

Comment:
I was thinking on something like this but inside the table, so all the search controllers are still there and the user can change the selection that took it into an empty set